cmd/open.go

```diff
@@ -20,9 +21,18 @@ func (ic ignoreClose) Close() error {
 	return nil
 }
 
+type ignoreCloseWithEventLogGetter struct {
```

Comment:
This was a hacky workaround in the original PR #128. Perhaps we instead add an option to pass a `TcgEventLog` field in `client.AttestOpts` and then fill that in directly with GetEventLog if it doesn't exist.